eodag_labextension/config.py

```diff
@@ -7,11 +7,20 @@
 from __future__ import annotations
 
 from importlib.metadata import distributions
+from typing import Optional
 
 from pydantic import BaseModel, computed_field
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
 
+class MapInfo(BaseModel):
+    """Map info model"""
+
+    tile_url: Optional[str] = None
+    tile_copyright: Optional[str] = None
+    zoom: Optional[int] = None
```

Comment:
Use `zoom_offset` instead of `zoom` to better reflect the behavior of the parameter. `zoom` would more reflect to the default zoom level while `zoom_offset` describes the zoom level shift to align the map zoom with the tile zoom.

```suggestion
    zoom_offset: Optional[int] = None
```